examples/skr/aci-arm-template.json

```diff
@@ -30,7 +30,16 @@
               "command": [
                 "/skr.sh"
               ],
-              "image": "mcr.microsoft.com/aci/skr:2.2",
+              "environmentVariables": [
+                {
+                  "name": "LogFile",
+                  "value": "/log.txt"

```

Comment:
You can't do this as an environment variable without some, likely excessively hard, policy to make sure the log doesn't get splatted over some file we rely on.

A basic rule of thumb is to never expose anything that gives the outside control over internal UVM/container state.

Reply:
If it is expressed as an environment variable, it would have to be captured in the security policy.  So long as the policy doesn't allow ".*", this would be safe against malicious changes.